### DISCUSSION TITLE
es/python/getting-started/starting-a-new-project/

### DISCUSSION THREAD

**Discussion by giscus** [dogweather/forkful#40] [Page Discussions]
# es/python/getting-started/starting-a-new-project/

Cómo hacerlo: Un entorno virtual es un directorio autocontenido que contiene todos los ejecutables necesarios para usar los paquetes que un proyecto…

https://forkful.ai/es/python/getting-started/starting-a-new-project/

**Comment by batwin2005** [comment]
from flask import Flask
app = Flask(__name__)

@app.route("/")
def hello_world():
    return "<p>Hola, Mundo!</p>"

if __name__ == "__main__":
    app.run(debug=True)
